microscopy_SEM001/sub-01/sub-01_sessions.tsv

```diff
@@ -0,0 +1,3 @@
+session_id	acq_time
+ses-01	2015-11-04T14:10:00
+ses-02	2015-11-04T14:15:00
```

Comment:
The present microscope Manufacturer (Hamamatsu) and Model (C4742-95-12G04) is a SEM one, I put it to the SPIM by inadvertance, I will change it for the *UltraMicroscope II* by *Miltenyi Biotec*, it seems like a SPIM microscope. 

I will add the metadata fields `SampleStaining` and `stain` for the SPIM dataset, no problem. I will comment down below when I'll have those plausible informations.

Reply:
After a quick research, I think the luxol fast blue stain (LFB) would be plausible for SPIM.
As it's used for light/optical microscopy ([Luxol fast blue stain, Wikipedia](https://en.wikipedia.org/wiki/Luxol_fast_blue_stain)) and since SPIM microscopy is a fluorescence microscopy technique ([Light sheet fluorescence microscopy, Wikipedia](https://en.wikipedia.org/wiki/Light_sheet_fluorescence_microscopy)), I think it could work.

I will commit those changes and push them to the branch!

Reply:
You can use the label `LFB` in the filename (`_stain-LFB_`) and the string `"Luxol fast blue"` in `SampleStaining`.